source/codegen/metadata/nitclk/config.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# This file is generated from NI-TClk API metadata version 255.0.0d0
+config = {
+    'api_version': '255.0.0d0',
```

Comment:
Make this match the version of the released version of the niTClk.h header you pulled in. You say in your PR description that you used 20.7. Did you get the header by installing 20.7 on a machine and copying the file? Did you copy it from a specific exported location in source control? Is 20.7 a released version?

Reply:
The `api_version` field actually comes from [nimi-python](https://github.com/ni/nimi-python/blob/master/src/nitclk/metadata/config.py) metadata and points to version that was used for generating nimi-python metadata. Should we be updating that?

- We do document changes we are making w.r.t. nimi-python in CHANGES.md (so essentially differences with this version)
- niTClk.h has constants defined for major/minor version that the header file belongs to. So 20.7 is document there.

> Did you get the header by installing 20.7 on a machine and copying the file? 

Yes. Tclk is not available as separate installer, but if you install niScope, that installs TClk as dependency. And we copied the header file it installs with that. Version of TClk installed was 20.7 in this case.

Reply:
And yes, 20.7 is released version of TClk.

Reply:
@jonlugoatx  since we have other PRs waiting on this PR before moving to owners, I am going to complete this PR and leave this comment open. If you think we should update version number, we can do that in one of the follow up PRs.

Reply:
If the "255.0.0d0" came from nimi-python then I'm not too worried. The more important thing is that 20.7 is released.